package.json

```diff
@@ -14,6 +14,7 @@
     "pm2:prod": "cross-env pm2 start ts-node -- -P tsconfig.json ./src/server.ts --node-args='--production'",
     "watch": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec 'ts-node' src/index.ts",
     "lint": "eslint src/**/**/*.ts && eslint tests/**/**/*.ts",
+    "lint-fix": "eslint --fix src/**/**/*.ts && eslint tests/**/**/*.ts",
```

Comment:
I think this isn't necessary to be added. Can `yarn lint --fix` can replace this?

Reply:
The command `yarn lint --fix` can't fix the lint error in the ts file under `src` folder (/**/*.ts).
This is because `--fix` option only applies to `eslint tests/**/**/*.ts`, which is the half of whole `yarn lint` command.

So, I fixed the PR commit again as below

(before)
```
"lint": "eslint src/**/**/*.ts && eslint tests/**/**/*.ts",
```
->
<br />
(after)
```
"lint": "eslint src/**/**/*.ts tests/**/**/*.ts",
```

Now, `--fix` option works for the files under both `src` and `tests` folders